clarifai/utils/misc.py

```diff
@@ -242,7 +242,7 @@ def parse_github_url(self, url: str) -> Tuple[str, str, str, str]:
             return owner, repo, branch, folder_path
 
         except Exception as e:
-            raise ValueError(f"Failed to parse GitHub URL: {e}")
+            logger.error(f"Failed to parse GitHub URL: {e}")
```

Comment:
After this codeflow needs to be break
```suggestion
            logger.error(f"Failed to parse GitHub URL: {e}")
            sys.exit(1)
```